frontend/src/utils/data.ts

```diff
@@ -64,6 +64,7 @@ export function summaryStatisticsByServiceAreaAndCensus(
   censusCategoryGroups.forEach(censusGroup => {
     statisticsByGroup[censusGroup] = summaryStatistics(rps, adequacies, { censusCategory, censusGroup })
   })
+  statisticsByGroup[' Total Population'] = summaryStatistics(rps, adequacies)
```

Comment:
is there supposed to be a space at the beginning of this string?

Reply:
yep, it's a trick to have it appear first